source/ocsp-support/ocsp-support.rst

```diff
@@ -134,6 +134,42 @@ Drivers SHOULD validate OCSP Responses in the manner specified in `RFC
 6960: 3.2 <https://tools.ietf.org/html/rfc6960#section-3.2>`__ to the
 extent that their TLS library allows.
 
+Suggested OCSP Caching Behavior
+-------------------------------
+Drivers with sufficient control over their TLS library's OCSP
+behavior SHOULD implement an OCSP cache. The key for this cache
+SHOULD be the certificate identifier (CertID) as specified in
+`RFC 6960: 4.1.1
+<https://tools.ietf.org/html/rfc6960#section-4.1.1>`__.
+For convenience, the relevant section has been duplicated below\:
+
+.. code:: ASN.1
+
+   CertID          ::=     SEQUENCE {
+       hashAlgorithm       AlgorithmIdentifier,
+       issuerNameHash      OCTET STRING, -- Hash of issuer's DN
+       issuerKeyHash       OCTET STRING, -- Hash of issuer's public key
+       serialNumber        CertificateSerialNumber }
+
+If a driver would accept a conclusive OCSP response (stapled or
+non-stapled), the driver SHOULD cache that response. We define a
+conclusive OCSP response as an OCSP response that indicates that a
+certificate is either valid or revoked. Thus, an unknown certificate
+status SHOULD BE considered conclusive and the corresponding OCSP
```

Comment:
I think this is supposed to say "SHOULD NOT"

Reply:
Already fixed! You started to review faster than I could push apparently :)